source/lex.tex

```diff
@@ -68,22 +68,11 @@
 \indextext{character!source file}%
 \indextext{character set!basic source}%
 Physical source file characters are mapped, in an
-\impldef{mapping physical source file characters to basic source character set} manner,
-to the basic source character set (introducing new-line characters for end-of-line
-indicators) if necessary.
+\impldef{mapping physical source file characters to translation character set} manner,
```

Comment:
@jensmaurer, this is subtly different from the other cross-reference for the same concept (the other index entry has "mapping from" instead of just "mapping").